crates/api/src/module.rs

```diff
@@ -199,8 +207,8 @@ impl Module {
             _ => None,
         }
     }
-    pub fn validate(_store: &Store, binary: &[u8]) -> bool {
-        validate(binary, None).is_ok()
+    pub fn validate(_store: &Store, binary: &[u8]) -> Result<()> {
+        validate(binary, None).map_err(|e| Error::new(e))
```

Comment:
The test errors are present because we are using default wasmparser config (`None`). The config shall be configured to accept multi-value.